portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro

```diff
@@ -1058,7 +1058,18 @@ definition {
 				value1 = "${fieldPlaceholder}");
 		}
 
-		if (isSet(fieldHelp)) {
+		if (isSet(fieldsetModal) && isSet(fieldHelpTextEdit)) {
```

Comment:
hey @Odrakir1 , could you please change the logic to:

```
if(isSet(fieldHelp)){
if(isSet(fieldsetModal)){
-> do what you implemented ...
}else{
-> do the existing implementation of (isSet(fieldHelp))
}
```

Also, don't use/create a different variable (`fieldHelpTextEdit`) to the same purpose of `fieldHelp`.